runtime/lib/backend/lightning/lightning_kokkos/LightningKokkosSimulator.hpp

```diff
@@ -24,7 +24,7 @@
 #include <stdexcept>
 
 #include "AdjointJacobianKokkos.hpp"
-#include "LinearAlgebra.hpp"
+#include "LinearAlgebraKokkos.hpp"
```

Comment:
Thanks! We used to rely on some of the helper functions in LinearAlgebra module for pos-processing the host state-vector in the runtime. But this is not required anymore! @vincentmr Did this cause any issue with building LK+CUDA? :thinking: 

We can remove this header
```suggestion
```